relay/network_graph/dijkstra_weighted.py

```diff
@@ -127,6 +127,15 @@ def get_capacity(u, v, data):  # gets the capacity from u to v
             "node %s not reachable from %s" % (source, target))
 
 
+def get_balance(G, a, b):
```

Comment:
Why do we put this outside of "find_path_triangulation" if we do not use it elsewhere?